Dockerfile

```diff
@@ -1,47 +1,36 @@
-#--------- Generic stuff all our Dockerfiles should start with so we get caching ------------
-FROM ubuntu:trusty
-MAINTAINER Tim Sutton<tim@linfiniti.com>
+FROM ubuntu:14.04
+MAINTAINER admire@afrispatial.co.za
 
-RUN  export DEBIAN_FRONTEND=noninteractive
-ENV  DEBIAN_FRONTEND noninteractive
-RUN  dpkg-divert --local --rename --add /sbin/initctl
-#RUN  ln -s /bin/true /sbin/initctl
 
-# Use local cached debs from host (saves your bandwidth!)
-# Change ip below to that of your apt-cacher-ng host
-ADD 71-apt-cacher-ng /etc/apt/apt.conf.d/71-apt-cacher-ng
+#ADD 71-apt-cacher-ng /etc/apt/apt.conf.d/71-apt-cacher-ng
 
-RUN echo "deb http://archive.ubuntu.com/ubuntu trusty main universe" > /etc/apt/sources.list
 RUN apt-get -y update
-# socat can be used to proxy an external port and make it look like it is local
-RUN apt-get -y install ca-certificates socat openssh-server supervisor
-RUN mkdir /var/run/sshd
-ADD sshd.conf /etc/supervisor/conf.d/sshd.conf
-
-
-RUN echo 'root:geogit' | chpasswd
-#-------------Application Specific Stuff ----------------------------------------------------
-
-# Next line a workaround for https://github.com/dotcloud/docker/issues/963
-RUN apt-get install -y --no-install-recommends openjdk-7-jdk
+RUN apt-get -y install  default-jdk
+RUN apt-get  install -y wget unzip daemontools
+RUN mkdir -p  /etc/service
+RUN mkdir -p  /etc/service/geogig_serve
+WORKDIR /etc/service/geogig_serve
+
+RUN touch run
+RUN echo "#!/bin/bash" >>run
```

Comment:
using the > to create an empty file results in the container not working at all.